src/screens/ui/RootNavigator.tsx

```diff
@@ -89,21 +97,15 @@ export default () => {
       <Stack.Screen
         name="Applets"
         options={{
-          title: 'Applets',
           headerStyle: {
             backgroundColor: colors.lighterGrey2,
           },
           headerTitleStyle: {
             fontSize: 16,
             color: colors.tertiary,
           },
-          headerLeft: () => (
```

Comment:
Can we leave it temporarilly in order to change user if we don't have logout yet?
Or just to have possibility to go to login screen

Reply:
We are implementing logout feature today, so no worries